18-additional-resources.Rmd

```diff
@@ -76,15 +76,17 @@ Liu, M. C., & Huang, Y. M. (2017). The use of data science for education: The ca
 
 Rosenberg, J. M., Lawson, M. A., Anderson, D. J., Rutherford, T., & Jones, R. S. (accepted pending minor revisions). Making Data Science “Count”: Data Science and Learning, Design, and Technology Research. In E. Romero-Hall (Ed.), Research Methods in Learning Design & Technology. Routledge: New York, NY.
 
+Dutt, A., Ismail, M. A., & Herawan, T. (2017). A systematic review on educational data mining. IEEE Access, 5, 15991-16005.
+
 ## Programming with R
 
 Wickham, H. & Grolemund, G. (2017). *R for data science*. Sebastopol, CA: O'Reilly.
 
-> A new classic, with a focus on accessible tools.
+> Learn how to use R to turn raw data into insight, knowledge, and understanding. This book introduces you to R, RStudio, and the tidyverse, a collection of R packages designed to work together to make data science fast, fluent, and fun. Suitable for readers with no previous programming experience, R for Data Science is designed to get you doing data science as quickly as possible.
 
 Teetor, P. (2011). *R cookbook*. Sebastopol, CA: O'Reilly.
 
-> 
+> This book provides over 200 practical recipes for analysing data using R. 
```

Comment:
can you mention what is meant by recipes here? Just less than one sentence?

Reply:
thank you for the feedback. I have updated the text.

Reply:
Thanks, good call.